snuba/query/accessors.py

```diff
@@ -0,0 +1,10 @@
+from typing import Set
+
+from snuba.query.expressions import Column, Expression
+
+
+def get_columns_in_expression(exp: Expression) -> Set[Column]:
+    """
+    Returns all the columns referenced in an arbitrary AST expression.
+    """
+    return set(e for e in exp if isinstance(e, Column))
```

Comment:
This sounds like it would return a complete list of columns from any expression (i.e. nested as well) but it doesn't seem to do that

Reply:
Fancy